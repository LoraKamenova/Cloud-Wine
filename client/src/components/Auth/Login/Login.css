.login-form-wrapper > .form-header-wrapper > .form-header-heading-container > .form-header-heading {
    bottom: -25px;
    left: 90px
}

.form-wrapper.login-form-wrapper {
    height: 60%;
    margin: auto;
}

