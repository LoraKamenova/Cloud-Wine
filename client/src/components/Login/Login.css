.login-page {
    background-image: url('https://res.cloudinary.com/dyhmxus4n/image/upload/v1615655024/React%20Project/login_dbmoom.jpg');
    height: 100vh;
    display: flex;
    flex-direction: column;
}

.login-form-wrapper {
    display: flex;
    flex-grow: 1;
    width: 40%;
    height: 80%;
    /*margin-top: 4%;*/
    /*margin-bottom: 4%;*/
    /*margin-right: 7%;*/
    margin: 4% auto;
    padding: 20px 30px;
    align-self: flex-end;
    justify-content: center;
}
