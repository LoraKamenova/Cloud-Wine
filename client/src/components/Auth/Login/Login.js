import './Login.css'
import FormHeader from "../../Other/FormHeader/FormHeader";

const Login = () => {
    return (
        <div className="custom-section login-section">

            <div className="login-container">

                <div className="auth-form-wrapper login-form-wrapper">

                    <FormHeader title="Вход"/>

                    <form className="auth-form" action="" method="">

                        <div className="auth-form-field-wrapper">
                            <div className="auth-form-field-heading">
                                <label htmlFor="username">Потребителско име:</label>
                            </div>
                            <input className="auth-form-field-input" type="text" id="username" name="username"/>
                        </div>

                        <div className="auth-form-field-wrapper">
                            <div className="auth-form-field-heading">
                                <label htmlFor="password">Парола:</label>
                            </div>
                            <input className="auth-form-field-input" type="password" id="password" name="password"/>
                        </div>

                        <div className="auth-form-button-wrapper">
                            <button className="auth-form-button" type="submit">ИЗПРАТИ</button>
                        </div>
                    </form>
                </div>
            </div>
        </div>
    );
};

export default Login;
