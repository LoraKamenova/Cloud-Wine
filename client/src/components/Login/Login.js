import './Login.css'

const Login = () => {
    return (
        <main className="login-reg-pages login-page">

            <section className="form-wrapper login-form-wrapper">

                <div className="form-site-name-wrapper">
                    <div className="form-site-name-text">
                        <h1 className="form-site-name">Винoписи</h1>
                    </div>
                    <div className="form-site-name-pic">
                        <img className="form-pic"
                             src="https://res.cloudinary.com/dyhmxus4n/image/upload/v1615712293/React%20Project/grape_border_ogd8gq.png"/>
                    </div>
                </div>

                <form className="form" action="" method="">
                    <div className="form-heading-wrapper">
                        <h1 className="form-heading">ВХОД</h1>
                    </div>

                    <div className="form-field-wrapper">
                        <div className="form-field-heading">
                            <label htmlFor="username">Потребителско име:</label>
                        </div>
                        <input className="form-field-input" type="text" id="username" name="username"/>
                    </div>

                    <div className="form-field-wrapper">
                        <div className="form-field-heading">
                            <label htmlFor="password">Парола:</label>
                        </div>
                        <input className="form-field-input" type="password" id="password" name="password"/>
                    </div>

                    <div className="form-button-wrapper">
                        <button className="form-button" type="submit">ИЗПРАТИ</button>
                    </div>
                </form>
            </section>
        </main>
    );
};

export default Login;
