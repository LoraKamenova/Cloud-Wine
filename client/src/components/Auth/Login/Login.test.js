import { render } from '@testing-library/react';
import Login from './Login';

describe('Login Component', () => {

    it("Login Component snapshot", () => {
        const { asFragment } = render(<Login />);
        expect(asFragment()).toMatchSnapshot();
    });

});
