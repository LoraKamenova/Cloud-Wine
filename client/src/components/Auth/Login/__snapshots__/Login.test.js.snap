// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Component Login Component snapshot 1`] = `
<DocumentFragment>
  <section
    class="form-wrapper login-form-wrapper"
  >
    <div
      class="form-header-wrapper"
    >
      <div
        class="form-header-heading-container"
      >
        <h1
          class="form-header-heading"
        >
          Вход
        </h1>
      </div>
      <div
        class="form-header-image-container"
      >
        <img
          alt="vine"
          class="form-header-image"
          src="https://res.cloudinary.com/dyhmxus4n/image/upload/v1615712293/React%20Project/grape_border_ogd8gq.png"
        />
      </div>
      <h1
        class="form-header-heading-hidden"
      >
        Вход
      </h1>
    </div>
    <form
      class="form-container"
    >
      <div
        class="form-field-wrapper"
      >
        <div
          class="form-field-heading"
        >
          <label
            for="username"
          >
            Потребителско име:
          </label>
        </div>
        <input
          class="form-field-input"
          id="username"
          name="username"
          type="text"
          value=""
        />
      </div>
      <div
        class="form-field-wrapper"
      >
        <div
          class="form-field-heading"
        >
          <label
            for="password"
          >
            Парола:
          </label>
        </div>
        <input
          class="form-field-input"
          id="password"
          name="password"
          type="password"
          value=""
        />
      </div>
      <div
        class="form-error-div"
      />
      <div
        class="form-button-wrapper"
      >
        <input
          class="form-button auth-form-button"
          type="submit"
          value="ИЗПРАТИ"
        />
      </div>
    </form>
  </section>
</DocumentFragment>
`;
